packages/fuel-gauge/src/e2e-script.test.ts

```diff
@@ -76,6 +77,28 @@ describe.each(selectedNetworks)('Live Script Test', (selectedNetwork) => {
     wallet = new WalletUnlocked(privateKey, provider);
   });
 
+  it('can deploy a large contract to a live Fuel Node', async () => {
```

Comment:
@petertonysmith94 I've added an e2e for validation but tbh I'm not sure whether we should merge it with this. It seems unnecessarily costly and the only additional proving it gives is to validate blob deploys. With the same logic we should be e2e testing everything. Our current e2es prove compatibility with the live network for a subset of functionality and i think that's enough.

Reply:
@danielbate I agree - doesn't seem wise to run on every e2e test run. Thanks for proving the change against the network though, that's valuable in itself.

Reply:
Note to remove the e2e before resolving this conversation.

Reply:
Is it worth just skipping - rather than removing entirely?